src/types/DataStructureTypes.tsx

```diff
@@ -31,6 +31,12 @@ interface ITemporal {
   revision?: string;
 }
 
+interface ICitation {
```

Comment:
Can we move all these back to the store, we should have defined a OGCCollection with variable and possible access. Aim is to remove the DataStructureTypes.tsx